tests/lbs_py_override_unittest.py

```diff
@@ -232,15 +232,12 @@ def testCompare(self):
         self.assertTrue(size1 <= size1)
 
         self.assertTrue(size1 > None)
-        self.assertTrue(size1 >= None)
-        self.assertFalse(size1 < None)
         self.assertFalse(size1 <= None)
         self.assertFalse(size1 == None)
         self.assertTrue(size1 != None)
 
         size3 = Size(0)
         self.assertTrue(size3 > None)
-        self.assertFalse(size3 < None)
```

Comment:
*coverity* doesn't know that ``Size`` is actually a very weird class than may (and used to) not work properly when comparing to ``None``.